pvt/helpers/src/models/tokens/TokenList.ts

```diff
@@ -160,4 +160,8 @@ export default class TokenList {
       this.tokens.sort((tokenA, tokenB) => (tokenA.address.toLowerCase() > tokenB.address.toLowerCase() ? 1 : -1))
     );
   }
+
+  scaleBalances(rawBalances: number[]): BigNumber[] {
```

Comment:
What if instead we called this `scaledBalances` and had it receive a function that returns a number, and then call it repeateadly and scale them? e.g.

```
scaledBalances(rawBalance: (number) => BigNumber)): BigNumber[] {
   return this.tokens.map((t, i) => rawBalance(i).mul(10 ** t.decimals));
}
```

We would then simplify the callsites to e.g. `initialBalances: poolTokens.scaledBalances(() => 10 + random(10))`